earthpy/spatial.py

```diff
@@ -57,9 +58,7 @@ def extent_to_json(ext_obj):
 
 
 def normalized_diff(b1, b2):
```

Comment:
How about
`Take two n-dimensional numpy arrays and calculate the normalized difference.`

`Math will be calculated (b1-b2) / (b1 + b2). `